### PR TITLE
Agdigger 222

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**JIRA**
https://issues.jboss.org/browse/AGDIGGER-222

As part of the linked JIRA saw some small issues that can be fixed quickly.

### CODE REVIEW

File: java/tasks/main.yml [hunk 1]
```diff
@@ -3,13 +3,7 @@
 -
   include: linux.yml
   when: ansible_distribution != 'MacOSX'
-  tags:
```
Comment (PhilipGough) [inline java/tasks/main.yml]: Potentially change this to `ansible_distribution == 'CentOS ' or ansible_distribution == 'Red Hat Enterprise Linux'` if its not working on Fedora

Comment (laurafitzgerald) [reply java/tasks/main.yml]: @PhilipGough yes will do that. I've also added a note to this JIRA [1] as suggested by @aliok but I think it's possible to make this change quickly.

[1] https://issues.jboss.org/browse/AGDIGGER-216

**Comment by aliok** [comment]
@laurafitzgerald I am curious about this commit: https://github.com/aerogear/digger-installer/pull/99/commits/1c8a974d9e6c7627ab0a1bf0638192753eb231dd

Can you elaborate please?

**Comment by laurafitzgerald** [comment]
@aliok it is the same as this [change](https://bluejeans.com/5258180441) without it the deploy tag cannot be used. To skip a task you have to provide all tags related to the task for it to be skipped.

**Comment by laurafitzgerald** [comment]
Merging

**Pull request merged**